test/sharness/t0054-dag-car-import-export.sh

```diff
@@ -9,44 +9,52 @@ export -f ipfsi
 set -o pipefail
 
 tar -C ../t0054-dag-car-import-export-data/ --strip-components=1 -Jxf ../t0054-dag-car-import-export-data/test_dataset_car_v0.tar.xz
+tab=$'\t'
 
 reset_blockstore() {
-  node=$1
-  ipfsi $1 pin ls --quiet --type=recursive | ipfsi $1 pin rm &>/dev/null
-  ipfsi $1 repo gc &>/dev/null
+  node=$1; shift
```

Comment:
Do we need the shift?

Reply:
Just symmetry with the next function. I will remove it.